nekoyume/Assets/_Scripts/BlockChain/ActionUnrenderHandler.cs

```diff
@@ -257,14 +257,25 @@ private void ResponseDailyReward(ActionBase.ActionEvaluation<DailyReward> eval)
                 return;
             }
 
+            if (GameConfigStateSubject.ActionPointState.ContainsKey(eval.Action.avatarAddress))
+            {
+                GameConfigStateSubject.ActionPointState.Remove(
+                    eval.Action.avatarAddress);
+            }
```

Comment:
Is this OK to write same logic with [here](https://github.com/planetarium/NineChronicles/pull/275/files#diff-69bc228a7e488d68e730931b3afb9790b5726c653cf8389c20b8e047b66fdc0eR214-R218)?

The unrender logic is commonly reverse the render logic, like below I guess.

```suggestion
            if (!GameConfigStateSubject.ActionPointState.ContainsKey(eval.Action.avatarAddress))
            {
                GameConfigStateSubject.ActionPointState.Add(
                    eval.Action.avatarAddress, bool);
            }
```

Reply:
이거 언랜더불리면 actionpoint 사용하는게 취소되서 로딩이 안뜨게 되야합니다 고로 remove가 맞습니더

Reply:
완전 헷갈리기 쉬운 부분이네요. 랜더, 언랜더는 무조건 반대로 작성해 왔어서요. ㅎㅎ 이부분 되돌리면서 관련 주석 달아 주시기로 했어요. cc @Unengine